src/platform/utilities/feature-toggles/featureFlagNames.json

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
if PR looks ok after any more changes, I'll make sure to update to latest featureFlagNames in `main` before re-generating the JSON. I'm basically just importing the old js module and calling `JSON.stringify` on it, so it can match the latest config from main whenever we merge